inspirehep/migrator/utils.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""Migrator utils."""
+
+
+from dojson.contrib.marc21.utils import create_record
+from flask import url_for
+from inspire_utils.helpers import force_list
+
+REAL_COLLECTIONS = (
+    "INSTITUTION",
+    "EXPERIMENT",
+    "JOURNALS",
+    "JOURNALSNEW",
+    "HEPNAMES",
+    "HEP",
+    "JOB",
+    "JOBHIDDEN",
+    "CONFERENCES",
+    "DATA",
+)
+
+
+def get_collection(marc_record):
+    collections = set()
+    for field in force_list(marc_record.get("980__")):
+        for v in field.values():
+            for e in force_list(v):
+                collections.add(e.upper().strip())
+    if "DELETED" in collections:
+        return "DELETED"
+    for collection in collections:
+        if collection in REAL_COLLECTIONS:
+            return collection
```

Comment:
Maybe instead last for loop do:
```suggestion
            collections &= set(REAL_COLLECTIONS)
            return collections.pop() if collections else "HEP"
```
WDYT?